src/index.html

```diff
@@ -6,9 +6,24 @@
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Moyo header</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&amp;display=swap&amp;subset=cyrillic" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Moyo header</h1>
+  <header class="header">
+  <img src="./images/logo.png" class="main-logo" alt="logo-error">
+  <nav class="nav-container">
+    <ul class="nav-list">
+      <li class="nav-item"><a class="nav-link is-active" href="">APPLE</a></li>
+      <li class="nav-item"><a class="nav-link" href="">SAMSUNG</a></li>
+      <li class="nav-item"><a class="nav-link" href="">СМАРТФОНЫ И ТЕЛЕФОНЫ</a></li>
+      <li class="nav-item"><a class="nav-link" data-qa="hover" href="">НОУТБУКИ И КОМПЬЮТЕРЫ</a></li>
+      <li class="nav-item"><a class="nav-link" href="">ГАДЖЕТЫ</a></li>
+      <li class="nav-item"><a class="nav-link" href="">ПЛАНШЕТЫ</a></li>
+      <li class="nav-item"><a class="nav-link" href="">ФОТО</a></li>
+      <li class="nav-item"><a class="nav-link" href="">ВИДЕО</a></li>
```

Comment:
One tag - one line. Use new lines for nested tags. Don't write nested tags in one line.